client/src/App.js

```diff
@@ -39,12 +39,10 @@ function App() {
     of a JSX element will be read as text like HTML and not as JavaScript. */
   return (
     <div className="App">
-      {/* <header className="App-header"> */}
-
       <header className="App-header">
```

Comment:
One thing we'll want to fix here is that the `header` tag should end before `main`. It often just contains a navbar. So our semantic layout will have 3 main sections, `header`, `main`, and `footer`